File: app/src/test/java/com/sumian/sleepdoctor/util/TimeUtilTest.java
package com.sumian.sleepdoctor.util;

import android.text.format.DateUtils;

import com.sumian.sleepdoctor.utils.TimeUtil;

import org.junit.Test;

/**
 * <pre>
 *     author : Zhan Xuzhao
 *     e-mail : xuzhao.z@sumian.com
 *     time   : 2018/6/1 10:47
 *     desc   :
 *     version: 1.0
 * </pre>
 */
public class TimeUtilTest {
    private long currentTime() {
        return System.currentTimeMillis();
    }

    @Test
    public void test() {
        System.out.println(TimeUtil.formatDate("M月d日", currentTime() + DateUtils.DAY_IN_MILLIS * 12));
    }
}

File: doctor/src/test/java/com/sumian/sddoctor/util/TimeUtilTest.kt
package com.sumian.sddoctor.util

import org.junit.Test

/**
 * <pre>
 *     @author : Zhan Xuzhao
 *     e-mail : xuzhao.z@sumian.com
 *     time   : 2018/6/20 20:26
 *     desc   :
 *     version: 1.0
 * </pre>
 */
class TimeUtilTest {
    @Test
    fun test() {
        System.out.println(TimeUtil.formatDate("yyyy-MM-dd", 1529510400000L))
    }
}